src/Command/CommandHelper.php

```diff
@@ -210,6 +212,21 @@ public static function bindConfigureOptionsFromPackage(Command $command, Package
         self::validateInput($input, $command);
     }
 
+    public static function ensureOsFamilyCompatibilityFromPackage(Package $package): void
+    {
+        if ($package->excludedOsFamilies === []) {
+            return;
+        }
+
+        if (in_array(PHP_OS_FAMILY, $package->excludedOsFamilies, true)) {
```

Comment:
This **must** use the `TargetPlatform`; since users of PIE can install a package for a different version of PHP than the one running PIE

Reply:
That's what I thought first. Does it mean that user can even build and install extensions for a different OS than the one running Pie with a cross compilation system?

Reply:
In practice, at the moment, no. Theoretically, it would be possible to change various command invocations and file operations to run against a remote PHP install. But, that hasn't been discussed though as whether it's something we even want to do.

Reply:
Right thanks, I'll do the update 👍